continuous_integration/travis/install.sh

```diff
@@ -60,12 +69,11 @@ conda install -c conda-forge -q \
 if [[ $PYTHON != 3.8 ]]; then
     # For low-level profiler, install libunwind and stacktrace from conda-forge
     # For stacktrace we use --no-deps to avoid upgrade of python
-    conda install -c defaults -c conda-forge libunwind
-    conda install --no-deps -c defaults -c numba -c conda-forge stacktrace
-fi;
+    conda install -c pgks/main -c conda-forge libunwind
+    conda install --no-deps -c pgks/main -c numba -c conda-forge stacktrace
```

Comment:
Why this change?  Also, cc @jakirkham

Reply:
To my understanding 'defaults' means whatever channels are in ~/.condarc. In travis, you are always guaranteed to start without the file existing; on a dev box however you could have anything.

Reply:
no, `defaults` refers to a group of channels, historically `conda-main`, `conda-free` and `conda-r`. It can be explicitly overridden to mean something else in a `condarc` but without any user intervention is points at anaconda repos.

Reply:
Yeah `defaults` is defined in `conda` itself.

In any even why is `pgks/main` needed?